spec/swagger.yaml

```diff
@@ -0,0 +1,82 @@
+swagger: '2.0'
+schemes:
+    - https # Remove if your API donsn't support HTTPS
+host: citrination.com
+basePath: /api
+info:
+    title: Citrination
+    version: 1.0.0
+    termsOfService: 'https://citrination.com/tos'
+    contact:
+        name: Citrine Informatics
+        email: admin@citrine.io
+        url: 'https://citrine.io'
+    license:
+        name: Apache License Version 2.0
+        url: 'https://www.apache.org/licenses/LICENSE-2.0'
+    description: |
+        # Introduction
+        The Citrination API provides much of the functionality that is available through the Citrination web interface. In addition, there exists some functionality that is only available through the API (for example, some of the more advanced query elements). All access to the API is controlled using a unique token assigned to each user (see the section on authentication).
+
+        Users may access the API directly through https requests, or may use one of the client libraries described below.
+
+        # Python client
+        A python client is available through [github](https://github.com/CitrineInformatics/python-citrination-client) or can be installed with pip using `pip install citrination-client`. There may be issues installing libraries that depend on [six](https://pypi.python.org/pypi/six) on OSX. If you encounter issues during installation, you may also try `pip install --ignore-installed six citrination-client`. The python client can be configured using:
+
+        ```python
+        from citrination-client import *
```

Comment:
needs to be an underscore: `citrination_client`

Reply:
good catch